cdk-apigw-authorizer-sqs/README.md

```diff
@@ -0,0 +1,81 @@
+# CDK Python project for deploying Queue-Based Leveling pattern!
+
+This pattern also referred to as Queue-Based leveling pattern can be useful when desigining Event driven architectures (EDA). Customers facing performance issues, when processing synchronous and webhook call, during extremely high peak traffic can avoid such situations by adapting this architectural strategy. A SQS queue will acts as a buffer for these synchronous requests and permits you to decouple the sender from the receiver. This pattern offers an easy solution to bridge the gap using AWS services that can easily plugin to customers existing architecture.
+
+![Alt text](images/apigw-arch.png?raw=true "Pattern using API gateway, SQS and Lambda Authorizer")
+
+
+This CDK project is written in Python, it has 2 stacks and can synthesize 2 cloud formation templates. Customers can deploy either of the stacks based on their API authorization needs:
+ 
+##### Stack 'ApigwSqsAuthStack' will create:
+* API Gateway Rest API with a POST method acting as a proxy to SQS,
+* A Token Authorizer for the API Gateway ( Lambda ARN of the function implementing the authorization is provided as a input parameter to CF and CDK)
+* Amazon SQS queue to store the payload posted
+
+##### Stack 'APigwSqsStack' will create:
+* API Gateway Rest API with a POST method acting as a proxy to SQS,
+* Amazon SQS queue to store the payload posted
+
+
+Important: this application uses various AWS services and there are costs associated with these services after the Free Tier usage - please see the [AWS Pricing page](https://aws.amazon.com/pricing/) for details. You are responsible for any AWS costs incurred. No warranty is implied in this example.
+
+## Requirements
+
+* [Create an AWS account](https://portal.aws.amazon.com/gp/aws/developer/registration/index.html) if you do not already have one and log in. The IAM user that you use must have sufficient permissions to make necessary AWS service calls and manage AWS resources.
+* [Git Installed](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git)
+* [CDK Installed](https://docs.aws.amazon.com/cdk/v2/guide/getting_started.html) 
+* [Create Lambda authorizer from blueprint](https://github.com/awslabs/aws-apigateway-lambda-authorizer-blueprints/blob/master/blueprints/python/api-gateway-authorizer-python.py)
+
+![Alt text](images/lambda-auth-blueprint.png?raw=true "Create lambda function using blueprint")
+
+
+## Deployment Instructions
+
+1. Create a new directory, navigate to that directory in a terminal and clone the GitHub repository:
+```
+git clone https://github.com/aws-samples/serverless-patterns/cdk-apigw-authorizer-sqs.git
+```
+2. Change directory
+```
+cd cdk-apigw-authorizer-sqs
+```
+3. To generate a cloudformation templates (optional)
```

Comment:
Can you take the region from the configuration and make CDK create the queue name. You can also use a config file inside the app to pass the parameters as this is not a clean way to do the call. 
Same for the deploying